src/eko/msbar_masses.py

```diff
@@ -328,52 +334,71 @@ def evolve(
             )
             matching = 1.0
             for pto in range(1, strong_coupling.order[0]):
-                for l in range(pto + 1):
+                for logpow in range(pto + 1):
```

Comment:
```suggestion
                for logpow in range(1, pto + 1):
```
This should fix the `0**0` issue without changing the thing, since all the `m_coeffs[0, :]` are 0

Reply:
This is not what I observe, since if L is 0, the only contributions to `matching` are coming from the `logpow=0` term...

Reply:
yes, true. I was wrong `pto=0` is already excluded, we do the same for `a_s`. But then yes this is what we wanted...
```suggestion
                # 0**0=1, from NNLO there is a matching also in this case
                for logpow in range(pto + 1):
```